### PR TITLE
docs: Update CONTRIBUTING.md to explain signing DCO

### PR DESCRIPTION
**Pull request by tsteenbe**
As several new contributors have asked how to sign the DCO.

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,8 +2,26 @@
 
 The team behind the [OSS Review Toolkit](https://github.com/heremaps/oss-review-toolkit) gratefully accepts contributions via
 [pull requests](https://help.github.com/articles/about-pull-requests/) filed against the
-[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls). As part of filing a pull request we ask you to
-sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO).
+[GitHub project](https://github.com/heremaps/oss-review-toolkit/pulls).
+
+# Signing each Commit
```
Comment (sschuberth) [inline CONTRIBUTING.md]: Nit: Blank line after this one.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,8 +2,26 @@
...
+As part of filing a pull request we ask you to
+sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) in each commit.
```
Comment (sschuberth) [inline CONTRIBUTING.md]: "sign off the" should still do to the previous line.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,8 +2,26 @@
...
+
+A DCO is lightweight way for a contributor to confirm that they wrote or otherwise have the right to submit code or documentation to a project.
```
Comment (sschuberth) [inline CONTRIBUTING.md]: Please hard-warp lines at column 120.

**Comment by sschuberth** [comment]
> As several new contributors have asked how to sign the DCO.

I'm surprised this is an issue, as next to the "DCO" check there's a "Details" link at leads to https://probot.github.io/apps/dco/, which explains it all.

File: CONTRIBUTING.md (hunk 2 above)
Comment (tsteenbe) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 3 above)
Comment (tsteenbe) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sschuberth) [reply CONTRIBUTING.md]: It does not seem to be done 😉

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,8 +2,30 @@
...
+# Signing each Commit
+
+As part of filing a pull request we ask you to
+sign off the [Developer Certificate of Origin](https://developercertificate.org/) (DCO) in each commit.
+Any Pull Request with commits that are not signed off will be reject by the
+[DCO check](https://probot.github.io/apps/dco/).
+
+A DCO is lightweight way for a contributor to confirm that they wrote or otherwise have the right
+to submit code or documentation to a project.
+Simply add `Signed-off-by` as shown in the example below to indicate that you agree with the DCO.
```
Comment (sschuberth) [inline CONTRIBUTING.md]: This sentence should also already start at line 14 as in the output no paragraph / line break is created unless also a blank line is added. I.e. making this sentence start at line 14 makes the visual appearance of this text and rendered output more similar, which is a good thing.

**Comment by tsteenbe** [comment]
> I'm surprised this is an issue, as next to the "DCO" check there's a "Details" link at leads to https://probot.github.io/apps/dco/, which explains it all.

Not only an issue for new ORT contributors but from discussions with other OSS project maintainers seems people new to contributing do not fully understand how to resolve failing PR checks. 

> Anyway, maybe you should also add a link to https://probot.github.io/apps/dco/ from the text, saying that we use Probot to perform the check?

Added an one-liner mentioning Probot as "DCO check"

**Comment by sschuberth** [comment]
> do not fully understand how to resolve failing PR checks.

Ah, ok. So it's not an issue specific to our project. 👍

File: CONTRIBUTING.md (hunk 4 above)
Comment (sschuberth) [reply CONTRIBUTING.md]: Ping...

File: CONTRIBUTING.md (hunk 1 above)
Comment (tsteenbe) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (tsteenbe) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,8 +2,30 @@
...
+A DCO is lightweight way for a contributor to confirm that they wrote or otherwise have the right
+to submit code or documentation to a project. Simply add `Signed-off-by` as shown in the example below
+to indicate that you agree with the DCO.
+
+An example signed commit message:
+
+```
+    spdx-utils: Add sanity checks for values of the mapping objects
+
+    Signed-off-by: John Doe john.doe@oss-review-toolkit.org
```
Comment (mnonnenmacher) [inline CONTRIBUTING.md]: Email should be in angle brackets: `Signed-off-by: John Doe <john.doe@oss-review-toolkit.org>`

Comment (tsteenbe) [reply CONTRIBUTING.md]: Done, please merge this PR when you are all OK with it.

**Pull request merged**